pages/slice/language-guide/class-types.md

```diff
@@ -178,6 +178,25 @@ When the generated code decodes a class in sliced format and slices off derived
 slices: it keeps them in the decoded instance. If you later send this class instance to another application that knows
 the derived class type, this application will decode successfully the full type.
 
+## Compact type IDs
+
+When the generated code encodes a class it needs to transmit the class's type.
+It does this by encoding the class's _type ID_ - a unique value associated with each class.
+Normally, this is the class's fully-qualified identifier.
+However, to reduce overhead, you can assign a numeric type ID to a class instead. For example:
+
+```slice
```

Comment:
I would do something simple like:

```slice
class Foo {} // no compact type ID
class Bar(42) // compact type ID of 42
```


```

Reply:
I added a module named `Foo` and changed the names to `ClassA` and `ClassB`.
I agree the extra long names didn't add anything.